app/views/user/emails/_email.html.slim

```diff
@@ -11,8 +11,11 @@
         = t('.primary')
 
   td
-    - unless email.primary?
+    - if !email.confirmed?
+      = t('.confirmation_sent')
+      = link_to t('.resend_confirmation'), send_confirmation_user_email_path(email), method: :post
```

Comment:
I had considered that, that doesn't tell user the email is sent already.  ...But it's OK.